app/models/post.rb

```diff
@@ -1,5 +1,14 @@
 class Post < ApplicationRecord
+  # belongs_to :user, class_name: 'user'
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
   belongs_to :user, class_name: 'User'
-  has_many :comments
-  has_many :likes
+
+  def update_user_post
```

Comment:
- [ ] You can use one of the `Rails Active Callbacks` - `after_save` to call the `#update_user_post` method every time a new post has been saved to the database, then make the `update_user_post` method private
e.g
```ruby
after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
Dear Ismailco,

Thank you for this feedback. I will implement this